### DISCUSSION TITLE
Unable to select multiple manual fields on pie chart (and others?)

### DISCUSSION THREAD

**Discussion by SharkWipf** [grafana/grafana#48706] [Dashboards, Panels & Visualizations]
**What happened**:  
Attempting to manually select fields on the "value options" section only allows picking a single field, which is pretty pointless for a pie chart: 
![image](https://user-images.githubusercontent.com/751205/166472234-bbb43532-eaa0-44cc-b361-c7aa6e3e7319.png)

**What you expected to happen**:
The ability to select multiple fields manually.

**How to reproduce it (as minimally and precisely as possible)**:
- Create a pie chart panel;
- Get data with multiple fields;
- Try manually specifying more than one field.

**Anything else we need to know?**:
The reason I need this is that I'm summing up multiple queries from the transform tab into one, and I don't need to display the original results, just the summed up ones.
Yes, modifying the query could bypass the problem, but I'd rather not have the queries deviate too much from the other panels in the dashboard.

**Environment**:
- Grafana version: v8.5.1 (61fe4a1859)
- Data source type & version: Postgres 14 (TimescaleDB)
- OS Grafana is installed on: Rootless Podman on Raspberry Pi OS
- User OS & Browser: Fedora, Chrome
- Grafana plugins: Whatever ships with the Docker image
- Others: Home Assistant Core

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
